### ISSUE TITLE
Link color contrast in Details List selected item

### ISSUE THREAD

**Issue by rwedoff** [microsoft/fluentui#19567]
### Environment Information

- **Package version(s)**: 8.30.0
- **Browser and OS versions**: Edge Chromium
### Describe the issue:

In the DetailsList component when a row is selected, a link element does not have sufficient color contrast for textual link information.

This can be found in the docs as well: https://developer.microsoft.com/en-us/fluentui?fabricVer=8#/controls/web/detailslist/customitemcolumns

![image](https://user-images.githubusercontent.com/6679636/131414204-bb4278a0-91bb-4637-a3c8-363433298bef.png)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/pen/?&prefill_data_id=9357c64a-e333-4fab-ad6f-0b3eafb91c09

#### Actual behavior:

The color contrast is 3.8:1

#### Expected behavior:

The color contrast should be 4.5:1

### Documentation describing expected behavior

https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

**Comment by gouttierre** [comment]
@rwedoff - Thanks for filing this issue with us. To set expectation our team will have a look at the issue but only once capacity allows.
  
@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by smhigley** [comment]
Fixed in a past PR, now links in hovered rows have a darker text color.
